CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
would also add cmake_policy(VERSION 3.16) too.
a tad bit redundant right now but for future purposes.

Reply:
added